test/testsymboldatabase.cpp

```diff
@@ -1880,6 +1888,29 @@ class TestSymbolDatabase: public TestFixture {
         }
     }
 
+    void functionDeclarations2() {
+        GET_SYMBOL_DB_STD("std::array<int,2> foo(int x);");
+
+        // 1 scopes: Global
+        ASSERT(db && db->scopeList.size() == 1);
+
+        if (db) {
```

Comment:
Such `if` was needed until 2-3 years ago. Nowadays an `ASSERT` will stop the test if the condition is false. So please remove this redundant `if`.

Reply:
I took the opportunity to clean the whole file.